server/db/benchmarkQueries.sql

```diff
@@ -0,0 +1,58 @@
+-- psql -d reviewsdb -f ./db/benchmarkQueries.sql
+\timing
+select r.*, p.id as photo_id, p.url from reviews r
+left join photos p on r.id = p.review_id
+where r.product_id = 63609;
```

Comment:
Eventually we'll want to benchmark how long it takes to fetch all / random IDs, since the database might be doing some caching if you run the same query multiple times ([ref](https://docs.google.com/document/d/1R0f_NmncL96rncDZiso_Eb6qhtUlA7WVl4XEXYQxijk/edit)).

My benchmarks are also hardcoded to the same `product_id`, so I need to update mine eventually too. But I think this is fine for now.

Reply:
updated query and query analysis.